### ISSUE TITLE
 Test with different betas and alphas

### ISSUE THREAD

**Issue by giovanni-br** [AlejandroUN/Neural-Style-Transfer-project-IMA206#2]
In the Gatys paper the loss function depends on the hyperparameters $\alpha$ and $\beta$:

$L_{\text{total}}(\hat{p}, \hat{a}, \hat{x}) = \alpha L_{\text{content}}(\hat{p}, \hat{x}) + \beta L_{\text{style}}(\hat{a}, \hat{x})$

We need to show  the influence of the ratio  $\frac{\alpha}{\beta}$ in the `.ipynb`
